action.yml

```diff
@@ -37,12 +40,12 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Login to registry - to fail in the beggining of action if creds are incorrect
-      uses: redhat-actions/podman-login@v1
-      with:
-        registry: ${{ inputs.registry }}
-        username: ${{ inputs.registry_username }}
-        password: ${{ inputs.registry_token }}
+#    - name: Login to registry - to fail in the beggining of action if creds are incorrect
```

Comment:
Why this is disabled? We need to be sure, that we have an access to registry and we are logged in.

Reply:
This is because of testing purposes.

Reply:
Will be changed when the PR is ready for merge.

Reply:
done